### PR TITLE
chore(vscode): add role ctx highlight

### PR DESCRIPTION
**Pull request by warmthsea**
在大型系统中，权限控制可能有数千个，我认为为权限控制添加特殊的 ide 颜色显示是不错的。

image

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] Please, don't make changes to `pnpm-lock.yaml` unless you introduce a new test example.

## Checklist

> ℹ️ Check all checkboxes - this will indicate that you have done everything in accordance with the rules in [CONTRIBUTING](contributing.md).

- [ ] If you introduce new functionality, document it. You can run documentation with `pnpm run docs:dev` command.
- [ ] Run the tests with `pnpm test`.
- [ ] Changes in changelog are generated from PR name. Please, make sure that it explains your changes in an understandable manner. Please, prefix changeset messages with `feat:`, `fix:`, `perf:`, `docs:`, or `chore:`.
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

## Summary by CodeRabbit

- **New Features**
  - Added a new Visual Studio Code extension for enhanced code color highlighting.
  - Introduced configuration settings for custom color highlighting on specific directives and file types.

- **Improvements**
  - Enhanced customization options for code visibility related to access control within the development environment.

### CODE REVIEW

**Comment by vince292007** [comment]
ci没过，合并下最新代码

File: .vscode/settings.json [hunk 1]
```diff
@@ -189,5 +189,15 @@
...
+  // 颜色高亮指令配置
+  "vscodeCustomCodeColor.fileLanguageIdList": ["vue", "typescript", "markdown"],
+  "vscodeCustomCodeColor.highlightValue": [
+    "v-access:code",
+    "v-access:role",
+    "hasAccessByCodes",
+    "hasAccessByRoles",
+    "AccessControl"
+  ],
+  "vscodeCustomCodeColor.highlightValueColor": "#3b82f6"
```
Comment (likui628) [inline .vscode/settings.json:202]: 是否可以使用vscode主题内的颜色

Comment (warmthsea) [reply .vscode/settings.json:202]: 不行，目前只支持特定颜色

**Comment by anncwb** [comment]
@warmthsea 这个我建议可以添加在文档即可

**Pull request closed without merging**